azure-functions/src/bindings/http_response.rs

```diff
@@ -106,7 +106,7 @@ impl HttpResponse {
     /// use azure_functions::bindings::HttpResponse;
```

Comment:
@peterhuene, there were no `#` preceding the import here. In other files, you have used the `#` when importing the module itself. Do you want me to add it here?

Reply:
Please do :) Let's go forward with that for consistency and correct it when we're touching places where the examples do it otherwise.

Reply:
As an edge-case, what about line 22 in this file where both Response and Request are imported? I would propose going from 

``` rust
use azure_functions::bindings::{HttpRequest, HttpResponse};
```

to 

``` rust
# use azure_functions::bindings::HttpResponse;
use azure_functions::bindings::HttpRequest;
```

Reply:
Sorry I didn't respond sooner.  Those examples at the type-level are more comprehensive, using the `#[func]` macro, for example.  I think we should probably leave those be, but for the method-level examples, we can hide the `use` statements.

Reply:
Cool, I just undid that commit.